### PR TITLE
create NiUserPane

### PR DESCRIPTION
**Pull request by nylira**
Create a user pane in the bottom of AppMenu that lets you:

1. Visit the user profile page (by clicking on the user avatar or name)
2. Edit your user settings (does nothing currently)
3. ee new and old notifications (does nothing currently)

screen shot 2017-11-27 at 11 36 11 am

I'm not sure if we will support avatars or usernames in the MVP, but if we don't they will be easy to change.

For now, clicking on the notification bell links to the "Welcome Screen", which I will work on tomorrow (following @jolesbi's wireframes).

If the user resizes the window to be shorter than the height of NiAppMenu, the main navigation links can be scrolled while NiUserPane appears on top:

screen shot 2017-11-27 at 11 43 20 am

### CODE REVIEW

**Comment by faboweb** [comment]
a comment for another time: native scrollbar looks pretty ugly on other OS
image

File: app/src/renderer/components/common/PageProfile.vue [hunk 1]
```diff
@@ -1,19 +1,18 @@
...
+    anchor-copy(value="TODO: user.pubkey" icon="content_copy")
     a(@click.native='signOut') Sign Out
 
   part(title='Profile')
-    list-item(dt="Total Vote Power" :dd="user.atoms" to="/vote-power")
+    list-item(dt="Total Vote Power" dd="TODO: user.atoms" to="/vote-power")
     list-item(dt="Solo Vote Power" dd="31.2M ATOM (19%)")
     list-item(dt="Delg. Vote Power" dd="33.8M ATOM (81%)")
     list-item(dt="Vote History" dd="37 Votes" to="/votes")
     list-item(dt="Proposals" dd="13" to="/proposals")
```
Comment (faboweb) [inline app/src/renderer/components/common/PageProfile.vue]: only link that "works"

Comment (nylira) [reply app/src/renderer/components/common/PageProfile.vue]: Just removed all the links that we don't have pages for yet.

**Comment by faboweb** [comment]
Scrollbar issue referenced in #118

**Pull request merged**